content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+  # same settings as runc
+  runtime_type = "io.containerd.runc.v2"
+  base_runtime_spec = "/etc/containerd/cri-base.json"
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler.options]
+    SystemdCgroup = true
+
+[plugins."io.containerd.grpc.v1.cri"]
+  # use fixed sandbox image
+  sandbox_image = "registry.k8s.io/pause:3.7"
+  # allow hugepages controller to be missing
+  # see https://github.com/containerd/cri/pull/1501
+  tolerate_missing_hugepages_controller = true
+  # restrict_oom_score_adj needs to be true when running inside UserNS (rootless)
+  restrict_oom_score_adj = false
+  
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin]
+  runtime_type = "/usr/local/bin/containerd-shim-spin-v2"
+
+```
+
+> NOTE: this file is a copy of the original one that can be found inside the KinD container. The only addition to the file is the declaration of the `spin` plugin (the last 2 lines)
+
+### KinD config file
+
+The second configuration file is related to KinD and will be used when creating a new cluster:
+
+- Create a file in your `$HOME` directory called `kind-spin.yaml` (for example)
+  - You can create it inside a directory, however it still should be located in your `$HOME` directory
+  - The location will be important when creating the KinD cluster
+
+**Windows Users ONLY**
+
+```yaml
+@"
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: /root/kind-spin/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+"@ | rdctl shell -- tee -a /root/test.yaml
+```
+
+**Linux and MacOS Users ONLY**
+
+```yaml
+cat <<EOF > $HOME/kind-spin.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: /home/${USER}/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+EOF
+```
+
+#### Connecting the dots (2)
+
+Rancher Desktop leverages 2 different technologies depending on the OS its installed.
+
+On Windows, WSL2 will be used and for Linux and MacOS, Lima is the preferred choice.
+
+While both technologies run Linux in a microVM, the behaviors differ in some parts. And the mountpoints with the host system are one of these differences.
+
+In the case of RD on WSL, the file generated is created **inside** the microVM, as nerdctl will need to have acceess to the file's path. Technically speaking, the mountpoint `/mnt/c` could also be used, however sometimes it's not available due to WSL main configuration. This way should be a bit more generic.
+
+Concerning RD on Lima, `$HOME/$USER` is mounted inside the microVM, therefore nerdctl will already see the files, and there's not need on copying the files over like it's done for WSL.
```

Comment:
```suggestion
Concerning RD on Lima, `$HOME` is mounted inside the microVM, therefore nerdctl will already see the files, and there's not need on copying the files over like it's done for WSL.
```

`$HOME` on mac/linux will be [`/Users` (Mac) or `/home` (Linux)]`/$USER`, right?

Reply:
Yes, you're totally right. As stated above, this error was due to a previous test I did and forgot to update it accordingly. Thanks a lot for pointing this out.

Reply:
Addressed in [1fd7d44](https://github.com/spinkube/documentation/pull/143/commits/1fd7d448b67ce170e5266d90849b190332e2015f)